CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Added
 
+- Added support for PVH boot mode.  This is used when an x86 kernel provides
+  the appropriate ELF Note to indicate that PVH boot mode is supported.
```

Comment:
Can you add a disclaimer here that for Linux kernels that were compiled with `CONFIG_XEN_PVH=y` this is a change in behavior, as they previously booted with LinuxBoot and will now boot with PVH (and that if people don't want that, they need to not set the compilation option)?

Reply:
I've added `Linux kernels compiled with CONFIG_XEN_PVH=y set this ELF Note` hopefully that's clear enough?

Reply:
Thanks, that should be good :)